server/resource_server/emails/verification/verifier_test.go

```diff
@@ -0,0 +1,152 @@
+package verification_test
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"globe-and-citizen/layer8/server/resource_server/emails/sender"
+	"globe-and-citizen/layer8/server/resource_server/emails/verification"
+	"globe-and-citizen/layer8/server/resource_server/emails/verification/code"
+	"globe-and-citizen/layer8/server/resource_server/models"
+	"globe-and-citizen/layer8/server/resource_server/utils/mocks"
+	"testing"
+	"time"
+)
+
+const adminEmail = "layer8@email.com"
+const userId uint = 1
+const username = "user"
+const userEmail = "user@email.com"
+const verificationCode = "12345"
```

Comment:
Can we have six digits here? Since currently we are doing six digits everywhere and also this test function `TestVerifyCode_VerificationCodeIsIncorrect` also has a six digit code.